views/sidebar.ejs

```diff
@@ -0,0 +1,22 @@
+<small> You'll receive your email subscriptions at your primary email <b><%emails[0]%></b> by default.</small>
+<hr style="height:20px;visibility:hidden">
```

Comment:
Also, bootstrap has a `small` class that does the same thing as `<small>` so you could get by with `<p class="small">`